### PR TITLE
[Code health] Use data binding for offline area download click handler

### PR DESCRIPTION
**Pull request by gino-m**
This was a small fix a made while debugging #811. @shobhitagarwal1612  PTAL?

### CODE REVIEW

File: gnd/src/main/java/com/google/android/gnd/persistence/mbtiles/MbtilesFootprintsParser.java [hunk 1]
```diff
@@ -36,15 +36,15 @@
 import org.json.JSONObject;
 import timber.log.Timber;
 
-public class GeoJsonParser {
+public class MbtilesFootprintsParser {
```
Comment (shobhitagarwal1612) [inline gnd/src/main/java/com/google/android/gnd/persistence/mbtiles/MbtilesFootprintsParser.java:39]: How about `MbtilesFootprintParser`?  (removed `s`)

**Review by shobhitagarwal1612 — APPROVED** [review]
One minor nit, otherwise LGTM.

My local setup isn't complete (app crashes due to incorrect SHA1) so wasn't able to verify the changes manually. Could you please confirm that you tested the changes locally?

File: gnd/src/main/java/com/google/android/gnd/persistence/mbtiles/MbtilesFootprintsParser.java (hunk 1 above)
Comment (gino-m) [reply gnd/src/main/java/com/google/android/gnd/persistence/mbtiles/MbtilesFootprintsParser.java]: Done!

**Comment by gino-m** [comment]
Done, @shobhitagarwal1612 PTAL!

**Review by shobhitagarwal1612 — APPROVED** [review]

**Pull request merged**